src/navigation/components/FullscreenMobileNavigation/FullscreenMobileNavigation.scss

```diff
@@ -0,0 +1,121 @@
+@import '../../../../styles/variables';
+@import '../../../../styles/mixins';
+
+$block: '.#{$ns}fullscreen-mobile-navigation';
+
+#{$block} {
+    position: fixed;
+    top: var(--header-height);
+    left: 0;
+    z-index: 100;
+
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: calc(100vh - var(--header-height) + 1px);
+
+    background-color: var(--g-color-base-background);
+    box-shadow: 0 3px 10px var(--g-color-sfx-shadow);
+
+    transition-timing-function: linear;
+    transition-property: left, right, top, bottom;
+    transition-duration: 300ms;
+
+    @include text-size(body-2);
+    @include mobile-tablet-only();
+
+    &__header,
+    &__content,
+    &__footer {
+        position: relative;
+        height: fit-content;
+        width: 100%;
+    }
+
+    &__content {
+        flex: 1 0 0;
+
+        padding-top: $indentXXS;
+        padding-bottom: $indentXXS;
+
+        overflow-y: auto;
+
+        scrollbar-width: none;
+        -ms-overflow-style: none;
+
+        &::-webkit-scrollbar {
+            display: none;
+        }
+
+        &_has-custom-content {
+            padding-top: 0;
+            padding-bottom: 0;
+        }
+    }
+
+    @media (max-width: map-get($gridBreakpoints, 'md') - 1) {
+        &__nav-lists {
+            padding-left: $indentXS;
+            padding-right: $indentXS;
+        }
+    }
+
+    &__button {
+        margin-top: $indentSM;
+    }
+
+    &__rows {
+        position: relative;
+
+        display: flex;
+        flex-direction: column;
+
+        padding-bottom: $indentSM;
+        @include reset-list-style();
+    }
+
+    &__rows:last-child {
+        margin-bottom: 0;
+    }
+
+    &__dropdown-item {
+        &:not(:last-child) {
+            margin-bottom: $indentXS;
+        }
+    }
+
+    &__popup {
+        @include mobile-tablet-only();
+        @include navigation-popup();
+    }
+
+    // * disable page scroll
+    body:has(&_opened) {
+        overflow-y: hidden;
+    }
+
+    // * appear directions
+    &_left {
+        left: -105vw;
```

Comment:
Can you please write in more detail why we use there are variables with viewport

Reply:
We decided to keep this logic, but rename the name of the prop